app/src/org/commcare/models/database/user/UserDatabaseUpgrader.java

```diff
@@ -580,6 +586,20 @@ private boolean upgradeTwentyOneTwentyTwo(SQLiteDatabase db) {
         }
     }
 
+    private boolean upgradeTwentyTwoTwentyThree(SQLiteDatabase db) {
+        db.beginTransaction();
+        try {
+            SqlStorage<DeviceReportRecord> existingStorage =
+                    new SqlStorage<>(DeviceReportRecord.STORAGE_KEY, DeviceReportRecordV1.class,
+                            new ConcreteAndroidDbHelper(c, db));
+            updateModels(existingStorage);
```

Comment:
@amstone326 The result of this is that the models have a uuid but it's empty, right? I'm a bit surprised that doesn't cause an issue, and it might be prudent to ensure that they get some value in the field if for no other reason than I don't think it's clear from the model that the field can be (blank? null?)